### ISSUE TITLE
No module named 'tkinter'

### ISSUE THREAD

**Issue by pslacerda** [schrodinger/pymol-open-source#137]
This errors happens when try to install a new plugin. I'm using the package `pymol` from the Ubuntu repositories.
```
ModuleNotFoundError: No module named 'tkinter'
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pymol/plugins/managergui_qt.py", line 297, in installplugin
    installPlugin(self)
  File "/usr/lib/python3/dist-packages/pymol/plugins/legacysupport.py", line 66, in installPlugin
    ofile = tkFileDialog.askopenfilename(title='Install Plugin',
  File "/usr/lib/python3/dist-packages/pymol/plugins/legacysupport.py", line 176, in __getattr__
    import tkinter.filedialog as module
ModuleNotFoundError: No module named 'tkinter'
```

**Comment by prcurran** [comment]
Try:
`import Tkinter`

**Comment by speleo3** [comment]
As far as I can tell, this was fixed in b5833292c38abddaa85c6584d89507449ab03193
